editoast/migrations/2024-09-06-114952_add-rolling-stock-freight-compatible/up.sql

```diff
@@ -0,0 +1,3 @@
+-- Your SQL goes here
+ALTER TABLE rolling_stock
+ADD freight_compatible BOOLEAN;
```

Comment:
Instead of making it nullable, wdyt about making it `DEFAULT FALSE`? (In the models only, having it `null | undefined` in the schemas and API can be convenient.)